debian/control

```diff
@@ -7,9 +7,9 @@ Build-Depends:
  debhelper (>= 9),
  dh-virtualenv (>= 1.0),
  git,
- python3 (>= 3.6),
- python3-venv (>= 3.6),
- python3-dev (>= 3.6),
+ python3.8 (>= 3.8),
+ python3.8-venv (>= 3.8),
+ python3.8-dev (>= 3.8),
```

Comment:
You have added `sofware-properties-common` in the build dependencies of the CI's job but not here. Is that intended?

Reply:
It was not used anymore so I removed it, thanks :)